user_management/ui/tests/test_views.py

```diff
@@ -76,3 +76,21 @@ def test_get_registered_user(self):
             self.view_class.already_verified_message,
             str(request._messages.store[0]),
         )
+
+    query_string = 'validated'
+
+    @override_settings(VALIDATED_QUERYSTRING=query_string)
```

Comment:
Can you change occurrences of validated to verified. I used the wrong words in the issue